support-workers/src/main/scala/com/gu/support/workers/lambdas/FailureHandler.scala

```diff
@@ -38,8 +38,9 @@ class FailureHandler(emailService: EmailService) extends FutureHandler[FailureHa
   private def sendEmail(state: FailureHandlerState) = {
     val emailFields = state.product match {
       case c: Contribution => FailedContributionEmailFields(email = state.user.primaryEmailAddress, IdentityUserId(state.user.id))
+      //TODO!!! will currently send nothing
```

Comment:
Is this TODO in the wrong place or is github just displaying things weirdly?

Reply:
That's also true (digital pack failure emails not being sent) but I admit not particularly relevant to this PR.